src/App.js

```diff
@@ -1,180 +1,20 @@
 import React from 'react';
 import './App.scss';
+import { MovieCard } from './components/MovieCard/MovieCard';
+import { MoviesList } from './components/MoviesList/MoviesList';
 // import moviesFromServer from './api/movies';
 
 export const App = () => (
   <div className="page">
     <div className="page-content">
       <div className="movies">
-        <div className="card">
-          <div className="card-image">
-            <figure className="image is-4by3">
-              <img
-                // eslint-disable-next-line max-len
-                src="https://m.media-amazon.com/images/M/MV5BMjAxMzY3NjcxNF5BMl5BanBnXkFtZTcwNTI5OTM0Mw@@._V1_SX300.jpg"
-                alt="Film logo"
-              />
-            </figure>
+        {MoviesList().map(movie => (
+          <div className="card" key={movie.id}>
+            <MovieCard
+              {...movie}
+            />
```

Comment:
```suggestion
        <MoviesList />
```
Тут должен быть просто компонент MoviesList